Zilon.Core/Zilon.Core.DiTests/Commands/MoveCommandTests.cs

```diff
@@ -112,9 +111,9 @@ public void CanRepeate_MonsterInSign_ReturnsFalse()
         public void CanRepeate_MonsterNotInSign_ReturnsTrue()
         {
             // ARRANGE
-            var command = Container.GetInstance<MoveCommand>();
-            var playerState = Container.GetInstance<ISectorUiState>();
-            var sectorManager = Container.GetInstance<ISectorManager>();
+            var command = ServiceProvider.GetRequiredService<MoveCommand>();
+            var playerState = ServiceProvider.GetRequiredService<ISectorUiState>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'playerState'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5012856)